docs/develop/plugins/publishing.md

```
@@ -30,15 +30,87 @@ To have your plugin start automatically after being installed, without requiring
+    "displayName": "My SK App",
+    "appIcon": "./img/icon-128.png",
+    "screenshots": [
+      "./docs/screenshots/main.png",
+      "./docs/screenshots/config.png"
+    ],
+    "requires": ["signalk-charts-plugin"],
+    "recommends": ["@signalk/freeboard-sk"]
   }
 ```
 
+#### `displayName`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)